azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachine.java

```diff
@@ -570,15 +571,17 @@
         /**
          * Specifies the time-zone.
          *
+         * @param timeZone the timezone
          * @return the stage representing creatable Windows VM definition
          */
         DefinitionWindowsCreatable withTimeZone(String timeZone);
 
         /**
          * Specifies the WINRM listener.
-         * <p/>
-         * Each call to this method adds the given listener to the list of VM's WinRM listeners.
+         * <p>
+         * Each call to this method adds the given listener to the list of VM's WinRM listeners
```

Comment:
actually, there should be a period after the sentence. 
(We don't need periods in param descriptions because they are not proper sentences)